lms/static/course_experience

```diff
@@ -0,0 +1 @@
+../../features/course_experience/static/course_experience
```

Comment:
Reviewers: this is a hack to get the LMS Karma tests to be able to run the course experience tests. This is the same solution that discussions used, so at least there is some precedent. We may instead want to have a separate Karma configuration for features, or maybe even one per feature.